portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,41 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Jennifer's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+    <header>
+      <h1>JENNIFER JIMENEZ</h1>
+    </header>
+    <nav>
+      <ul class="navbar">
+        <li>Home</li>
+        <li>Projects & Work</li>
+        <li>Gallery</li>
+      </ul>
+    </nav>
```

Comment:
Nice use of `header` and `nav` here! They make it easier to see the overall structure of the page.